compact/far_backward/lumi/spec_ScFi_cal.xml

```diff
@@ -0,0 +1,66 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2023 Aranya Giri -->
+
+<lccdd>
+        <!--
+        Note : Values/Notations for making modules pointing along z-axis.
+               Rotation of modules wrt XY axis after construction.          -->
+        <define>
+                <constant name="LumiSpecCAL_ModSizeX"            value="LumiSpecCAL_SiPMNumX*LumiSpecCAL_SiPMSizeXY"/>
+                <constant name="LumiSpecCAL_ModSizeY"            value="LumiSpecCAL_SiPMNumY*LumiSpecCAL_SiPMSizeXY"/>
+                <constant name="LumiSpecCAL_ModSizeZ"            value="LumiSpecCAL_DXY"/>
+                <constant name="LumiSpecCAL_FiberRadius"            value="0.25*mm"/>
+                <constant name="LumiSpecCAL_FiberSpaceX"            value="0.25*mm"/>
+                <constant name="LumiSpecCAL_FiberSpaceY"            value="0.25*mm"/>
+        </define>
+
+        <materials>
+        <material name="BrassAlloy360">
+                <D type="density" unit="g/cm3" value="8.49"/>
+                <fraction n="0.6045" ref="Cu"/>
+                <fraction n="0.3550" ref="Zn"/>
+                <fraction n="0.0370" ref="Pb"/>
+                <fraction n="0.0035" ref="Fe"/>
+        </material>
+        <material name="WPowderplusEpoxy">
+                <D type="density" unit="g/cm3" value="10.95"/>
+                <fraction n="0.97" ref="Tungsten"/>
+                <fraction n="0.03" ref="Epoxy"/>
+        </material>
+        </materials>
+
+        <display>
+                <vis    name="LumiSpecCAL_ModuleVis"            ref="AnlGold"   alpha="0.5"     visible="true" showDaughters="true"/>
+                <vis    name="LumiSpecCAL_FiberVis"             ref="AnlRed"    aplha="0.8"     visible="true"/>
+                <vis    name="LumiSpecCAL_FiberHolderVis"       ref="AnlBlue"   alpha="1.0"     visible="true"/>
+        </display>
```

Comment:
There are too many individual fibres and holders to reasonably visualise, these will have to default to invisible.

Reply:
Visualisation is kept this way for better understanding of CAL design, it can be changed accordingly in later phase.